documentation/release-notes/Stack/migration-guide.1.2.11.md

```diff
@@ -0,0 +1,117 @@
+# Table of Contents
+1. [Removal of Force parameters](#removal-of-force-parameters)
```

Comment:
I think you will want to add in the changes from the migration guides for 2.X and 3.X powershell for the modules you are using (Profile esp.): https://github.com/Azure/azure-powershell/blob/preview/documentation/release-notes/migration-guide.2.0.0.md, https://github.com/Azure/azure-powershell/blob/preview/documentation/release-notes/migration-guide.3.0.0.md, https://github.com/Azure/azure-powershell/blob/preview/documentation/release-notes/migration-guide.4.0.0.md